warehouse/query-core/src/main/java/datawave/query/discovery/DiscoveryLogic.java

```diff
@@ -71,12 +71,11 @@
     
     private static final Logger log = Logger.getLogger(DiscoveryLogic.class);
     
+    DiscoveryQueryConfiguration config;
```

Comment:
The base class already has a config member.  What we should do is use setConfig() on the base class instead and then override the getConfig() method to return a DiscoveryQueryConfiguration instead of the GenericQueryConfiguration.